src/components/DefinitionListItem.js

```diff
@@ -1,25 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ComponentFactory from './ComponentFactory';
+import { findKeyValuePair } from '../utils/find-key-value-pair';
 
-const DefinitionListItem = ({ nodeData: { children, term }, ...rest }) => (
-    <React.Fragment>
-        <dt>
-            {term.map((child, index) => (
-                <ComponentFactory nodeData={child} key={`dt-${index}`} />
-            ))}
-        </dt>
-        <dd>
-            {children.map((child, index) => (
-                <ComponentFactory
-                    {...rest}
-                    nodeData={child}
-                    key={`dd-${index}`}
-                />
-            ))}
-        </dd>
-    </React.Fragment>
-);
+const DefinitionListItem = ({ nodeData: { children, term }, ...rest }) => {
```

Comment:
Not sure if this one is being used in any articles? I looked for any `definitionList` directive but didn't see any in use, so this is likely fine :)

Reply:
Gotcha! It actually mostly appears in `.. glossary::` directives which I also don't see in devhub, so yup should be all good!